### ISSUE TITLE
no authentication smtp server doese not work

### ISSUE THREAD

**Issue by conan522** [thorsten/phpMyFAQ#3209]
We use a internal no authentication smtp server,  so I keep username/password blank, but looks does not work. Also “send test mail” button no response.

**Comment by thorsten** [comment]
@conan522 please check this fix

**Comment by conan522** [comment]
get error : Email "25" does not comply with addr-spec of RFC 2822.

**Comment by thorsten** [comment]
Did you try the "send test mail"?

**Comment by conan522** [comment]
Yes, but "send test mail" no any response. I try use "Contact", then got the error : Email "25" does not comply with addr-spec of RFC 2822.

**Comment by thorsten** [comment]
But you use a valid mail address, right?

**Comment by conan522** [comment]
Yes, I am sure I used a valid mail address.

**Comment by thorsten** [comment]
Is it an email address from your organization?

**Comment by thorsten** [comment]
@conan522 any feedback on this one?

**Comment by thorsten** [comment]
no feedback -> closed
